core/executor/runtime-test/src/lib.rs

```diff
@@ -5,48 +5,23 @@
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
-use rstd::{vec::Vec, slice, vec};
+#[cfg(not(feature = "std"))]
+use rstd::{vec::Vec, vec};
```

Comment:
I guess this is only needed under `no-std` because the macro below only expands under `no-std`. Would it make more sense to have the macro just use the full path to `rstd::vec::Vec` instead of relying on this import?

Reply:
The macro does not rely on this, the functions defined in this file depend on this. So, this is fine to include here.